cloud/ec2_fleet.go

```diff
@@ -303,26 +311,24 @@ func (m *ec2FleetManager) TerminateInstance(ctx context.Context, h *host.Host, u
 	}
 
 	for _, stateChange := range resp.TerminatingInstances {
-		if stateChange.InstanceId == nil {
```

Comment:
Why did you remove this one?

Reply:
It seems like it's looking for a condition that logically doesn't seem possibley. I looked through the Splunk logs and I've never seen this log before, so I think it's testing for a nil condition that never actually happens in practice. In any case, you could identify the issue indirectly from the `"terminated instance"` log if `instance_id` is empty.

FWIW I'm not sure this log gives us a lot of value since you could also just tell if the host was terminated in EC2 from the `"AWS API call"` logs.

Reply:
Ah sounds good! I thought it was probably unused but I was wondering if a change from the PR caused it to no longer be relevant, ty!